app/templates/events/view.hbs

```diff
@@ -1,6 +1,59 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if device.isMobile}}
+          <button class="ui button"><i class="unhide icon"></i></button>
```

Comment:
but this will remove the grouping of button for mobile though it looks good to me

<img width="393" alt="screen shot 2017-05-28 at 11 32 08 am" src="https://cloud.githubusercontent.com/assets/12716067/26526455/5b83df80-4399-11e7-8809-90f79dda06c1.png">

is this fine??